spec.html

```diff
@@ -28003,6 +28240,8 @@ <h2>Syntax</h2>
         ImportDeclaration :
           `import` ImportClause FromClause `;`
           `import` ModuleSpecifier `;`
+          `import` ImportClause FromClause [no LineTerminator here] WithClause `;`
```

Comment:
The `[no LineTerminator here]` is a left over from when the proposal used `assert` instead of `with`: since `assert` is a valid identifier, it was necessary to disambiguate this code:
```js
import "x"
assert
{}
```

As pointed out in https://github.com/tc39/proposal-import-attributes/issues/136, this restriction now can/should be removed. Is it a normative change that I need to present in plenary, or can I consider it as one of the small changes that happen during the pre-stage-3 reviews?

Reply:
You can make whatever normative changes you like without consensus prior to stage 3. I'd say it's good practice to call out any such changes when presenting so people have a chance to object, but IMO you do not need to ask for consensus for the change separately.